ci-operator/step-registry/openstack/conf/createfips/openstack-conf-createfips-commands.sh

```diff
@@ -4,8 +4,9 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml
+export OS_CLIENT_CONFIG_FILE=${SHARED_DIR}/clouds.yaml
 CLUSTER_NAME=$(<"${SHARED_DIR}"/CLUSTER_NAME)
+OPENSTACK_EXTERNAL_NETWORK=$(<"${SHARED_DIR}"/OPENSTACK_EXTERNAL_NETWORK)
```

Comment:
What is the purpose of this file? Aren’t you setting the same environment variable at the workflow level?

Reply:
transition. This patch is getting really big.